resources/lang/en/notifications.php

```diff
@@ -3,9 +3,12 @@
 return [
     'campaign'          => [
         'application'           => [
-            'approved'  => 'Your application to the :campaign campaign has been approved.',
+            'approved'  => 'Your application to the :campaign campaign has been approved. Message provided: :reason',
```

Comment:
By replacing the base key and adding a new key for the old translation, our translators won't know that the key needs to be updated. So to make their lives easier, please use the same key for approval, and create a new one for if there is a reason message :)